vhdbuilder/packer/post-install-dependencies.sh

```diff
@@ -1,20 +1,83 @@
 #!/bin/bash
+
+#time_stamps is an associative array that links the logical steps of the install script with timestamps
+#Logicalorder is a regular array that remembers the actual order of the steps (associative arrays are not indexed)
+declare -A time_stamps=()   
+declare -a logical_order=() 
+
+#The record_benchmark function adds the current step into logical_order and then associates that step with a start and end time in the timestamps array
+record_benchmark () { dt_stamp=$(date +%H:%M:%S); logical_order+=( "${1}" ); time_stamps["${1}"]=$dt_stamp; }
+
+#The PrintBenchmarks function uses the logical_order array as a control to chronologically iterate over the step/timestamp pairs in the timestamps array
+print_benchmark_results () { echo; echo; echo "Benchmarking Results:"; echo; for i in "${logical_order[@]}"; do echo "   $i: ${time_stamps[$i]}"; done; echo; echo; }
```

Comment:
Can we re-use the same helper functions across the files? To avoid copy-pasting.